### ISSUE TITLE
Cannot read property 'state' of undefined

### ISSUE THREAD

**Issue by vincent-cm** [MurhafSousli/ngx-progressbar#88]
It appears that one of the property is missing null check down the following code:

```
<ng-progress-bar
    [speed]="speed"
    [positionUsing]="positionUsing"
    [ease]="ease"
    [showSpinner]="showSpinner"
    [direction]="direction"
    [color]="color"
    [thick]="thick"
    [state]="progress.state | async" . ----- here the progress need a initial value or null check
  >
</ng-progress-bar>
```

**Comment by ogezue** [comment]
+1

**Comment by Sky4CE** [comment]
Same error:
![image](https://user-images.githubusercontent.com/9821633/31991154-ba9b307a-b97f-11e7-9bf9-147ff9d22001.png)

Angular 5.0.0-beta4 , using config For Http

**Comment by raj-optisol** [comment]
+1

**Comment by lucasmartinez** [comment]
same error

**Comment by TRAHOMOTO** [comment]
+1

**Comment by MurhafSousli** [comment]
Sorry! that was my mistake, fixed in 2.1.1
